apps/zui/src/plugins/brimcap/zeek/correlations.ts

```diff
@@ -29,7 +29,7 @@ export function activateZeekCorrelations() {
       return zedScript`
         from ${session.poolName} 
         | md5==${getMd5()} 
-        | count() by tx_hosts 
+        | count() by tx_host:=id.resp_h 
```

Comment:
I do the `:=` assignment here because if I leave it as just `id.resp_h` then only `id` appears as the column header in the table and no values under it, I guess since `id` is a record type and I assume the table is only prepared to render primitive values, and primitive values are all we want to show here anyway. So I'm kinda conjuring up `tx_host` and `rx_host` as fake field names just for presentation purposes, but that feels defensible.